shimmingtoolbox/coils/coordinates.py

```diff
@@ -125,3 +127,49 @@ def phys_to_vox_gradient(gx, gy, gz, affine):
              (gz * inv_affine[2, z_vox] * z_vox_spacing)
 
     return gx_vox, gy_vox, gz_vox
+
+
+def resample_from_to(nii_from_img, nii_to_vox_map, order=3, mode='constant', cval=0., out_class=nib.Nifti1Image):
```

Comment:
maybe change default order to 2 (trilinear), because 3 creates oscillations (maybe problematic for our applications?) and is longer